modules/pubsub/README.md

```diff
@@ -98,20 +98,20 @@ BigQuery subscriptions need extra configuration in the `bigquery_subscription_co
 ```hcl
 module "pubsub" {
   source     = "./fabric/modules/pubsub"
-  project_id = "my-project"
+  project_id = var.project_id
   name       = "my-topic"
   subscriptions = {
     test-bigquery = {
       bigquery = {
-        table               = "my_project_id:my_dataset.my_table"
+        table               = "${var.project_id}.${module.bigquery-dataset.dataset_id}.my_table"
```

Comment:
Two questions around here:
* should the separator between project and dataset name be `:` or `.`?
* Can we use `module.bigquery-dataset.tables` output to create output here? Will `id` or `self_link` for table work here?

Reply:
Thanks Wiktor, for the feedback.
- I think the `.` separator works as well as  illustrated in this  [ example ](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/pubsub_subscription).  But I'll keep the original implementation
-  Using outputs `id` or `self_link` is not enough. However, this works `"${module.bigquery-dataset.tables["my_table"].project}.${module.bigquery-dataset.tables["my_table"].dataset_id}.${module.bigquery-dataset.tables["my_table"].table_id}` or ` "${module.bigquery-dataset.tables["my_table"].project}:${module.bigquery-dataset.tables["my_table"].dataset_id}.${module.bigquery-dataset.tables["my_table"].table_id}` works

Reply:
Thanks, that works. It's not maybe nice, but follows the official provider docs.